tests/rptest/tests/cloud_storage_scrubber_test.py

```diff
@@ -536,9 +624,9 @@ def test_scrubber(self, cloud_storage_type):
 
         self._assert_metrics_updated()
 
-        #This test deletes segments, spillover manifests
-        #and fudges the manifest.rp - storage - tool also picks
-        #up on some of these things.
+        # This test deletes segments, spillover manifests
+        # and fudges the manifest.rp - storage - tool also picks
+        # up on some of these things.
```

Comment:
I think these were changed in a previous commit, so maybe rebase that away.

Reply:
Should be fixed.